src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -210,7 +206,7 @@ function ReplyButton({
         <img
           alt=""
           className={cssClass("replyButton--icon")}
-          src={require("./messages-black-icon.svg")}
```

Comment:
Looks like this icon isnt being used anywhere after this so we could remove it from the directory

Reply:
Good catch!